docs/ecosystem-roadmap.md

```diff
@@ -0,0 +1,80 @@
+
+This document is intended to provide a high-level overview of the Windows Console and Windows Terminal products, how they fit into the ecosystem of command-line applications across Windows and other operating systems, and a general roadmap of the products, features, and strategies that are a part of building the platform and building for this platform.
+
+## Definitions
+
+We'll start with some definitions of command words and phrases in this space that can help you understand the overall strategy and be used as reference throughout this document set.
+
+### Command Line Applications
+
+Command line applications, or sometimes called "console applications" and/or referred to as "clients" of the console subsystem, are programs that operate mainly on a stream of text or character information. They generally contain no user interface elements of their own and delegate both the output/display and the input/interaction roles to a hosting application. Command line applications receive a stream of text on their standard input `STDIN` handle which represents a user's keyboard input, process that information, then respond with a stream of text on their standard output `STDOUT` for display back to the user's monitor. Of course, this has evolved over time for additional input devices and remote scenarios, but the same basic philosophy remains the same: command-line clients operate on text and someone else manages display/input.
+
+### Standard Handles
+
+The standard handles are a series of handles, `STDIN`, `STDOUT`, and `STDERR`, that are introduced a part of process space on startup and represent a place where information can be accepted on the way in and sent back on the way out (including a special place to report errors out). For command-line applications, these must always exist when the application starts and are either inherited from the parent automatically, set explicitly by the parent, or created automatically by the operating system if neither of the first two are specified/permitted. For classic windows applications, these may be blank on startup, but can be implicitly or explicitly inherited from the parent or allocated, attached, and freed during runtime by the application itself.
+
+Standard handles do not imply a specific type of attached device, though in the case of command-line applications, the device is most commonly a console device, file (from redirection in a shell), or a pipe (from a shell connecting the output of one utility to the input of the next). It may also be a socket or any other type of device.
+
+### TTY/PTY
+
+On non-Windows platforms, the TTY and PTY devices represent respectively either a true physical device or a software-created pseudo-device that represent the same as a Windows console session: a channel where communication between a command-line client application and a server host interactivity application or physical keyboard/display device can exchange text-based information.

```

Comment:
`represent the same as` => `represent the same thing as`?

Reply:
"that are the same concept as"

given that I used the word represent twice in the sentence too